clients/admin-ui/src/features/system/SystemInformationForm.tsx

```diff
@@ -83,10 +85,27 @@ const SystemInformationForm = ({
     [passedInSystem, customFields.customFieldValues]
   );
 
+  const features = useFeatures();
+
   const [createSystemMutationTrigger, createSystemMutationResult] =
     useCreateSystemMutation();
   const [updateSystemMutationTrigger, updateSystemMutationResult] =
     useUpdateSystemMutation();
+  const { data } = useGetAllDictionaryEntriesQuery();
```

Comment:
nit: since this is a system form, can we rename `data` to something like `dictionaryData`? otherwise if I were skimming the file I'd assume `data` is system data.

Reply:
should we `skip` this get if we are in plus? or maybe only do it if the dictionary service is healthy?

Reply:
both great ideas. I'll add those in now